docs/UserGuide.md

```diff
@@ -69,31 +69,210 @@ The following annotated text-boxes are used throught this guide to provide usefu
 
 [Back to Table of Contents](#table-of-contents)
 
-## Quick start
-
-1. Ensure you have Java `17` or above installed in your Computer.
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` 
-   command to run the application.asd
-
-    {: .alert .alert-secondary}
-    > A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.
-    > 
-    > ![Ui](images/Ui.png)
-
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will 
-   open the help window.
-
-    {: .alert .alert-secondary}
-   > Some example commands you can try:
-   > * `list` : Lists all contacts.
-   > * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` tothe Address Book.
-   > * `delete 3` : Deletes the 3rd contact shown in the current list.
-   > * `clear` : Deletes all contacts.
-   > * `exit` : Exits the app.
-
-1. Refer to the [Features](#features) below for details of each command.
+# Quick Start
+
+In this section, lets learn how to set up MediBase3 on your computer.
+
+---
+
```

Comment:
![image](https://github.com/user-attachments/assets/179f4b6b-53c3-4d06-9fb7-dd1edde7e007)

remove this Horizontal line, I've already set the CSS to automatically add horizontal line for `## h2 header`s